packages/react/package.json

```diff
@@ -78,9 +79,9 @@
     "formik": "^2.2.5",
     "identity-obj-proxy": "3.0.0",
     "inquirer": "7.1.0",
-    "jest": "26.0.1",
-    "jest-axe": "4.1.0",
-    "jest-environment-jsdom-sixteen": "^1.0.3",
+    "jest": "^26.0.1",
```

Comment:
Wonder if we wish to include this '^' or let it be as it was before? There seems to be no common pattern for this. Though maybe defining explicit version is little more common?

Reply:
I can change that but it should be pretty safe to use the "hat". Then it would automatically use the latest minor or patch versions. But then again it might be risky if some library does not follow semver versioning strictly.

Reply:
I guess we need to use the hat to get the library updates and fixes. I found at least five library updates between the oldest major version and the latest one.